<launch>

  <arg name="gui" default="true" />

  <node name="image_publisher"
        pkg="jsk_perception" type="image_publisher.py">
    <rosparam subst_value="true">
      file_name: $(find jsk_perception)/sample/barkely_hide_bones.jpg
      encoding: rgb8
      publish_info: false
    </rosparam>
  </node>

  <node name="in_hand_recognition"
        pkg="jsk_perception" type="in_hand_recognition.py">
    <remap from="~input" to="image_publisher/output"/>
    <rosparam subst_value="true">
      images_path: $(find jsk_perception)/sample/in_hand_recognition_database_images.npy
      labels_path: $(find jsk_perception)/sample/in_hand_recognition_database_labels.npy
      gpu: -1
    </rosparam>
  </node>

</launch>

